skore/src/skore/utils/_accessor.py

```diff
@@ -59,6 +59,14 @@ def check(accessor: Any) -> bool:
         )
         if hasattr(estimator, "coef_"):
             return True
+        try:  # e.g. TransformedTargetRegressor()
+            if hasattr(estimator.regressor_, "coef_"):
+                return True
+        except AttributeError as msg:
+            if "object has no attribute 'regressor_'" in str(msg):
+                pass
+            else:
+                raise
```

Comment:
Maybe
```suggestion
            if "object has no attribute 'regressor_'" not in str(msg):
                raise
```
is the same?